src/runtime/components/forms/Input.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="ui.wrapper">
+  <div :class="(type === 'hidden') ? 'hidden' : ui.wrapper">
```

Comment:
The `ui.wrapper` contains the `relative` class for absolute icons underneath, did you try with icons?

Reply:
No, I did not style the input with icons.
As I mentioned in the description, when the `UInput` is part of (for example) a `div.grid.gap-4` the `div.relative` (ui.wrapper) of UInput will cause a gap between other elements - that's the root of my PR.

Reply:
But semantically you're absolutely right, there is not much sense of using a UI library for a hidden input-field - it just felt right for me to "fix" this 😅